IFC4x3/Properties/s/StaticDischargeHead_0BrAnLHer7kPgv2nOUi7ob/Documentation.md

```diff
@@ -0,0 +1 @@
+Static discharge head.
```

Comment:
Tough to get something useful from this description.

Reply:
"Measurement of the vertical distance from the discharge point to a reference elevation." ?